pyre/extra_tests/snippets/cli_file_null_bytes.py

```
@@ -0,0 +1,56 @@
+    startup = os.path.join(directory, "startup_nul.py")
+    with open(startup, "wb") as stream:
+        stream.write(b"x = '\0' trailing bytes\n")
+    env = os.environ.copy()
+    env["PYTHONSTARTUP"] = startup
+    result = subprocess.run(
+        [sys.executable, "-i"],
+        input=b"exit()\n",
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        env=env,
+    )
```

Ruff (0.16.2)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)